doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -556,7 +556,10 @@ Libraries for networking
 Libraries for NFC
 -----------------
 
-|no_changes_yet_note|
+* :ref:`nfc_t4t_isodep_readme` library:
+
+  * Fixed the ISO-DEP error recovery process when the R(ACK) frame is received in response to the R(NAK) frame from the poller device.
```

Comment:
```suggestion
  * Fixed the ISO-DEP error recovery process in case where the R(ACK) frame is received in response to the R(NAK) frame from the poller device.
```